CONTRIBUTING.md

```diff
@@ -62,6 +62,28 @@ Manually eye ball the list to make sure Issues are relevant to the release (Some
 
 Head to GitHub and author a new release add the changelog for the tag created.
 
+**Note**: Some manual steps are required before publishing new release as shown below.
+1. Download deliverables from Github release page 
+2. Remove VIB/DEB/RPMs from the ```Downloads``` sections
+3. Use internal Confluence page to sign the VIB (make sure to update [vDVS_bulletin.xml](https://github.com/vmware/docker-volume-vsphere/blob/master/docs/misc/vDVS_bulletin.xml#L19) to keep it current with the release
```

Comment:
Nit : Remove "(make sure to". 
Use internal Confluence page to sign the VIB (make sure to update -> Follow steps from internal Confluence page to sign the VIB. Update ....